src/docs/testing/ui-performance/index.md

```diff
@@ -293,25 +286,19 @@ Go to Flutter's timeline view in a browser as follows:
     icon][]), but the "view <u>inspector</u>" link takes you to the standard
     version of the timeline, not the version customized for Flutter.)
 
-    ![IntelliJ Flutter Inspector Window][]
-
-    [line-chart icon]: /docs/testing/ui-performance/images/observatory-timeline-icon.png
-    [stopwatch icon]: /docs/testing/ui-performance/images/observatory-icon.png
+    ![IntelliJ Flutter inspector window][]
 
  1. In VS Code, bring up the command palette and enter "observatory".
     Select **Flutter: Open Observatory Timeline** from the list that pops up.
     If this command isn’t available, make sure that the app is running.
 
-
 #### Using Observatory's timeline
 
 <aside class="alert alert-info" markdown="1">
-**Note:** The Observatory UI and Flutter's custom timeline page are currently
-evolving. For this reason, we aren't fully documenting the UI at this time.
-If you are comfortable experimenting with Observatory, and would like to give
-us feedback, please file [issues or feature
-requests](https://github.com/dart-lang/sdk/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-observatory)
-as you find them.
+  **Note:** The Observatory UI and Flutter's custom timeline page are currently
```

Comment:
I would remove mention of the observatory timeline or replace this with the 
devtools app timeline.